compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -933,6 +933,12 @@ error.invalid.interface.of.non.abstract.object=\
 error.unimplemented.referenced.method.in.class=\
   no implementation found for the method ''{0}'' of class ''{1}''
 
+error.unimplemented.referenced.method.in.service.declaration=\
+  no implementation found for the method ''{0}'' of service declaration ''{1}''
+
+error.unimplemented.referenced.method.in.object.constructor=\
+  no implementation found for the method ''{0}'' of object constructor ''{1}''
```

Comment:
Similar comment as above for this. Gives the feeling that the method is in the object constructor, not that the object constructor is expected to be providing an implementation for the method described in the contextually expected type.

Reply:
Something like, 
```
object constructor expected to provide an implementation for the method 'onMessage' of type 'SerT'
```